docs/sources/panels-visualizations/visualizations/xy-chart/index.md

```diff
@@ -0,0 +1,264 @@
+---
+canonical: https://grafana.com/docs/grafana/latest/panels-visualizations/visualizations/xy-chart/
+keywords:
+  - grafana
+  - chart
+  - xy chart
+  - documentation
+  - guide
+  - graph
+labels:
+  products:
+    - cloud
+    - enterprise
+    - oss
+description: Configure options for Grafana's xy chart
+title: XY chart
+weight: 100
+---
+
+# XY chart
+
+XY charts provide a way to visualize arbitrary x and y values in a graph so that you can easily show the relationship between two variables.
+
+![An xy chart showing height weight distribution](screenshot-xy-chart-v10.4.png)
+
+## Supported data formats
+
+You can use any type of tabular data in an xy chart. This type of visualization doesn't require time data.
+
+## XY chart options
+
+### Series mapping
+
+Set how series data is mapped in the chart.
+
+- [Auto](#auto-series-mapping-options) - Automatically generates series from all available data frames (or datasets). You can filter to select only one frame.
+- [Manual](#manual-series-mapping-options) - Explicitly define the series by selecting from available data frames.
+
+Depending on your series mapping selection, the **Frame**, **X-field**, and **Y-field(s)** options differ. The [Auto](#auto-series-mapping-options) and [Manual](#auto-series-mapping-options) series mapping sections describe those different settings.
+
+#### Auto series mapping options
+
+When you select **Auto** as your series mapping mode, the following options become available:
+
+- [Frame](#frame)
+- [X-field](#x-field)
+- [Y-fields](#y-fields)
+
+##### Frame
+
+By default, xy chart displays all data frames. You can filter to select only one frame.
+
+##### X-field
+
+Select which field or fields x represents. By default, this is the first number field in each data frame. For example, you enter the following CSV content:
+
+| a   | b   | c   |
+| --- | --- | --- |
+| 0   | 0   | 0   |
+| 1   | 1   | 9   |
+| 2   | 2   | 4   |
+
+In the resulting chart, x-fields are generated from the values in column "a" unless you define it differently.
+
+##### Y-fields
+
+The series of the chart are generated from the y-fields. After the x-field is set, by default, all the remaining number fields in the data frame are designated as the y-fields.
+
+You can select one y-field to include or you can use [overrides][Configure field overrides] to exclude y-fields individually. To disable y-fields individually, add an override with the following properties for each y-field you want removed:
+
+- Override type: **Fields with name**
+- Override property: **Series > Hide in area**
+- Area: **Viz**
+
+#### Manual series mapping options
+
+When you select **Manual** as your series mapping mode, the following options become available:
+
+- **Frame** - Select your data frame or dataset. You can add as many data frames as you want.
+- **X-field** - Select which field x represents.
```

Comment:
I'm not sure what we normally do in this situation but it could be helpful to emphasize `x` and `y` in the document. There were a few times I almost read over it but that may just be me

Reply:
Let's leave it for now, since at the moment, it's following our style rules. If we get feedback that indicates we need to make a change, we'll go from there.